Fastlane Secrets/Fastlane Secrets/AppDelegate.swift

```diff
@@ -22,8 +22,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         label.trailingAnchor.constraint(equalTo: viewController.view.safeAreaLayoutGuide.trailingAnchor).isActive = true
         label.topAnchor.constraint(equalTo: viewController.view.safeAreaLayoutGuide.topAnchor).isActive = true
 
-        label.text = "Secret key will go here!"
+        label.text = Secrets.apiKey
```

Comment:
Note the secrets file is not checked in to the repo, but could be generated using fastlane on a CI, for example